src/libs/CardUtils.ts

```diff
@@ -135,14 +135,21 @@ function maskCard(lastFour = ''): string {
  * Converts given 'X' to '•' for the entire card string.
  *
  * @param cardName - card name with XXXX in the middle.
+ * @param bank - card bank.
  * @returns - The masked card string.
  */
-function maskCardNumber(cardName: string): string {
-    if (!cardName || cardName === '') {
+function maskCardNumber(cardName: string, bank: string | undefined): string {
```

Comment:
NAB: I think we should put the param name is `feed` because I think it will be more correct meaning